lms/static/sass/_experiments.scss

```diff
@@ -366,29 +361,33 @@
 
 @media (min-width: 835px) {
     .v2.register-choice-certificate,
-    .v2.register-choice-donate {
+    .v2.register-choice-continue,
+    .v2.deco-divider {
         min-height: 250px;
     }
 }
 
 @media (min-width: 1024px) {
-    .v2 .donation-link {
+    .v2 .continue-link {
         width: 55%;
     }
     .v2.deco-divider .copy {
         margin-left: 15px;
     }
 }
 
-@media (min-width: 1064px) {
+@media (min-width: 1096px) {
     .v2.register-choice-certificate,
-    .v2.register-choice-donate {
+    .v2.register-choice-continue,
+    .v2.deco-divider {
         min-height: 260px;
     }
     .v2 .img-certificate, .v2 .img-donate {
+        margin-top: 10px;
         display: initial;
     }
-    .v2 .donation-link, .v2.register-choice-certificate button {
+    .v2 .continue-link, .v2.register-choice-certificate button,
+    .v2.register-choice-certificate input {
         margin-top: -22px !important;
     }
 }
```

Comment:
The Sass could be written more efficiently (e.g. so you only declare .v2 once and nest) but because this is just an experiment I'm fine with leaving it how it is for now.